src/python/WMCore/ReqMgr/Service/Request.py

```diff
@@ -404,26 +406,53 @@ def _handleNoStatusUpdate(self, workload, request_args, dn):
         """
         For no-status update, we only support the following parameters:
          1. RequestPriority
-         2. Global workqueue statistics, while acquiring a workflow
+         2. SiteWhitelist
+         3. SiteBlacklist
+         4. Global workqueue statistics, while acquiring a workflow
+        As Global workqueue statistics updates are exclusive to the rest of the
+        parameters. Meaning if it is about to be updated all the rest of the
+        request_args will be ignored.
         """
-        if 'RequestPriority' in request_args:
-            # Yes, we completely ignore any other arguments posted by the user (web UI case)
-            request_args = {'RequestPriority': request_args['RequestPriority']}
-            validate_request_priority(request_args)
-            # must update three places: GQ elements, workload_cache and workload spec
-            self.gq_service.updatePriority(workload.name(), request_args['RequestPriority'])
-            report = self.reqmgr_db_service.updateRequestProperty(workload.name(), request_args, dn)
-            workload.setPriority(request_args['RequestPriority'])
-            workload.saveCouchUrl(workload.specUrl())
-            cherrypy.log('Updated priority of "{}" to: {}'.format(workload.name(), request_args['RequestPriority']))
-        elif workqueue_stat_validation(request_args):
-            report = self.reqmgr_db_service.updateRequestStats(workload.name(), request_args)
-            cherrypy.log('Updated workqueue statistics of "{}", with:  {}'.format(workload.name(), request_args))
-        else:
-            msg = "There are invalid arguments for no-status update: %s" % request_args
+        report = []
+        reqArgsDiff = deepcopy(request_args)
+
+        if not reqArgsDiff:
+            cherrypy.log("Nothing to be changed at this stage")
+            return reduceReport(report)
```

Comment:
No, calling `reduceReport` on an empty list will return the expected value, because the expected value is set as a default.
```
In [10]: reduceReport([])
Out[10]: 'OK'
```